linters/export_delivery_data/getSprintData.graphql

```diff
@@ -0,0 +1,65 @@
+query ($endCursor: String, $login: String!, $project: Int!, $batch: Int!) {
+  # get the project by the organization login and project number
+  organization(login: $login) {
+    projectV2(number: $project) {
+      items(first: $batch, after: $endCursor) {
+        # allows us to use --paginate in the gh api call
+        pageInfo {
+          hasNextPage
+          endCursor
+        }
+        # fetch details per item in the list
+        nodes {
+          ... on ProjectV2Item {
+            content {
+              ...issueContent
+            }
+            sprint: fieldValueByName(name: "Sprint") {
+              ...iterationContent
+            }
+            pi: fieldValueByName(name: "Program Increment") {
+              ...iterationContent
+            }
+            points: fieldValueByName(name: "Points") {
```

Comment:
Maybe I'm reading this wrong, but isn't points an attribute of issue, rather than sprint? Same question for status.

Reply:
Is this question in reference to the comments in the `IssueMetadata` dataclass? If so, I updated the code comments to make it a bit more clear what I meant by "Sprint metadata" and "Roadmap metadata" in this commit: [docs: Improve python script documentation](https://github.com/agilesix/simpler-grants-sandbox/pull/43/commits/e0d1d3c9517eb4abdb682a75ff4c72be824fdf56)

The TL;DR is that yes, status and points are attributes _about_ the issue, but they only live on the _sprint board_ as opposed to attributes like type, closedAt, etc.

Reply:
After chatting about this in slack, we also decided to update the variable names for `pillar`, `status`, and `points` to make it easier to tell which entity this attribute is specific to:
- `deliverable_pillar`
- `issue_status`
- `issue_points`